apps/opencs/main.cpp

```diff
@@ -47,9 +47,6 @@ int runApplication(int argc, char *argv[])
     setenv("OSG_GL_TEXTURE_STORAGE", "OFF", 0);
 #endif
 
-    // To allow background thread drawing in OSG
-    QApplication::setAttribute(Qt::AA_X11InitThreads, true);
```

Comment:
Was this set to handle threading?

Won't we need this back when we fix or Qt5 issues with threading?

Reply:
> Was this set to handle the threaded renderer?

Yes, with Qt4. As I said, Qt5 does not use `XInitThreads()`, so this line just is no-op with Qt5.

Reply:
Great, thanks.